vllm/executor/ray_gpu_executor.py

```diff
@@ -161,6 +161,10 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
         self._run_workers("update_environment_variables",
                           all_args=all_args_to_update_environment_variables)
 
+        if len(node_gpus) == 1:
+            # in single node case, we don't need to get the IP address.
+            # localhost is sufficient
```

Comment:
nit: this is technically the loopback address; localhost is a dns mapping to the loopback address

Reply:
fixed in [d5bf0ea](https://github.com/vllm-project/vllm/pull/5619/commits/d5bf0eaa7b8952ee931819fea95671e3f6385541)